assets/js/bulk-move.js

```diff
@@ -140,49 +176,40 @@ jQuery(document).ready(function () {
 			return;
 		}
 
-		jQuery.ajaxSetup( { async: false } );
-
-		jQuery.post( ajaxurl, payload, function( response ) {
-
-			if ( ! response.success ) {
-				return;
-			}
-
-			var terms = response.data.terms || {};
-
-			if ( jQuery.isEmptyObject( terms ) ) {
-				alert( response.data.no_term_msg );
-
-				return;
-			}
-
-			jQuery( 'tr.term-select-row' ).show();
-
-			// Reset options on each AJAX request.
-			jQuery( '#smbm_mbct_selected_term, #smbm_mbct_mapped_term' ).children( 'option' ).remove();
-
-			jQuery( '<option/>', {
-				'value': '-1',
-				'text': response.data.select_term_label
-			}).appendTo( '#smbm_mbct_selected_term' );
-
-			jQuery( '<option/>', {
-				'value': '-1',
-				'text': response.data.remove_term_label
-			}).appendTo( '#smbm_mbct_mapped_term' );
-
-			jQuery.each( terms, function( termId, term ) {
-				jQuery( '<option/>', {
-					'value': termId,
-					'text': term['term_name']
-				}).appendTo( '#smbm_mbct_selected_term' );
-
-				jQuery( '<option/>', {
-					'value': termId,
-					'text': term['term_name']
-				}).appendTo( '#smbm_mbct_mapped_term' );
-			});
+		jQuery( 'tr.term-select-row' ).show();
+
+		jQuery( '#smbm_mbct_selected_term, #smbm_mbct_mapped_term' ).select2({
```

Comment:
Looks like some whitespace changes have gotten into. Revert those changes since they are adding noise to the diff.